docs/book/src/user/quick-start.md

```diff
@@ -1456,6 +1456,29 @@ helm install --kubeconfig=./capi-quickstart.kubeconfig --repo https://raw.github
 
 For more information, see the [CAPZ book](https://capz.sigs.k8s.io/topics/addons.html).
 
+{{#/tab }}
+{{#tab OpenStack}}
+
+Before deploying the Calico CNI, configure the `cloud.conf` file for integration with your OpenStack environment:
+
+```bash
+cat > cloud.conf <<EOF
+[Global]
+auth-url=<your_auth_url>
+application-credential-id=<your_credential_id>
+application-credential-secret=<your_credential_secret>
+region=<your_region>
+domain-name=<your_domain_name>
+EOF
+```
+
+Next, create a Kubernetes secret using this configuration to securely store your cloud environment details.
+You can create this secret for example with:
+
+```bash
+kubectl create secret generic capi-quickstart-cloud-conf --from-file=cloud.conf
```

Comment:
The name and namespace has to match what is used in the manifests
```suggestion
kubectl -n kube-system create secret generic cloud-config --from-file=cloud.conf
```
Ref https://github.com/kubernetes/cloud-provider-openstack/blob/64b813046f25b41aa4295a0a51726bcf25e92bc7/manifests/controller-manager/openstack-cloud-controller-manager-ds.yaml#L78-L80